File: FUNDAMENTAL JavaScript/JS Basic Syntax, Conditional Statements and Loops/printAndSum.js
function printAndSum(start,end){

    let sum = 0;
    let diff = "";

    for(i = start; i <= end; i++){

        sum += i;

        diff += "" + i + " ";


    }
    console.log(diff);
    console.log(`Sum: ${sum}`);


}
printAndSum(5, 10)

File: Java Fundamentel/src/basicSyntax/exercise/PrintAndSum.java
package basicSyntax.exercise;

import java.util.Scanner;

public class PrintAndSum {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int startNum = Integer.parseInt(scanner.nextLine());
        int endNum = Integer.parseInt(scanner.nextLine());

        int sum = 0;

        for (int i = startNum; i <= endNum; i++) {
        System.out.print(i + " ");
            sum += i;
        }
        System.out.println();
        System.out.println("Sum: " + sum);
    }

}
